Cubical/Categories/Monoidal/Base.agda

```diff
@@ -0,0 +1,121 @@
+-- Monoidal categories
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Monoidal.Base where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Constructions.Product
+open import Cubical.Categories.Functor.Base
+open import Cubical.Categories.Functor.Product
+open import Cubical.Categories.Morphism
+open import Cubical.Categories.NaturalTransformation.Base
+open import Cubical.Foundations.Prelude
+
+
+module _ {ℓ ℓ' : Level} (C : Category ℓ ℓ') where
+  open Category C
+
+  private
+    record TensorStr : Type (ℓ-max ℓ ℓ') where
+      field
+        ─⊗─ : Functor (C × C) C
+        i : ob
```

Comment:
`i` is very short... Maybe a slightly more descriptive name would be good?

Reply:
`unit`?

Reply:
Much better!